include/RAJA/policy/cuda/MemUtils_CUDA.hpp

```diff
@@ -318,19 +318,20 @@ size_t cuda_max_blocks(size_t block_size)
 
 struct CudaOccMaxBlocksThreadsData
 {
-  int prev_shmem_size;
+  size_t prev_shmem_size;
   int max_blocks;
   int max_threads;
 };
 
 template < typename RAJA_UNUSED_ARG(UniqueMarker), typename Func >
 RAJA_INLINE
-void cuda_occupancy_max_blocks_threads(Func&& func, int shmem_size,
-                                       size_t &max_blocks, size_t &max_threads)
+void cuda_occupancy_max_blocks_threads(Func&& func, size_t shmem_size,
```

Comment:
We should document conventions like this in the RAJA Dev Guide so we can try to keep type usage consistent during code reviews.